### PR TITLE
Added String() in constant.go.

### PR DESCRIPTION
**Pull request by hihiboss**
resolved:

details:

Implement String() in opcode.

~- [ ] Test case~

### CODE REVIEW

File: opcode/constant_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * limitations under the License.
+ */
+
+package opcode_test
+
+import "testing"
+
+// TODO: implement test cases :-)
+func TestType_ToString(t *testing.T) {
+
```
Comment (boohyunsik) [inline opcode/constant_test.go:23]: Are you implementing?

File: opcode/constant.go [hunk 2]
```diff
@@ -152,3 +154,38 @@ const (
 	// memory[offset:offset+32] = value
 	Mstore Type = 0x23
 )
+
+// Change the bytecode of an opcode to string.
+func (p Type) ToString() (string, error) {
```
Comment (AgwaB) [inline opcode/constant.go]: in golang, I know as `toSting()` should be `String()` !

File: opcode/constant_test.go (hunk 1 above)
Comment (hihiboss) [reply opcode/constant_test.go:23]: Nop! I'll leave it good first issue!

**Review by boohyunsik — APPROVED** [review]
Look great to me.
I will do compiler with you!

**Pull request merged**